build_scripts/generate_isotopes.js

```diff
@@ -41,8 +41,8 @@ for (let element in elements) {
       mainIsotope[1] = isotope;
     }
   }
-  if (ratioSum !== 1) {
-    throw new Error("Ratios do not add up to 1 for ".concat(element));
+  if ((Math.round(ratioSum*1000000)/1000000 !== 1) && !elements[element].disabled) {
```

Comment:
Good catch on the floating precision. I think this could be made more readable. What if we did something like:

let difference = 1-ratioSum;
if(Math.abs(difference) > 1e-6)...